.gitignore

```diff
@@ -93,3 +95,6 @@ ENV/
 
 # SQLite3
 db.sqlite3
+
+# migrations
+authors/apps/*/migrations/*
```

Comment:
We do not git ignore migrations files.

Reply:
Working on fixing the migrations errors that arise with that change. Thanks